### PR TITLE
Feature/save return

### PR DESCRIPTION
**Pull request by solidgoldpig**

### CODE REVIEW

File: metadata/config/service.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "_id": "service",
+  "_type": "config.service",
+  "dataRetentionDuration": 28,
+  "homepageUrl": "https://www.gov.uk",
+  "name": "Service name goes here",
+  "phase": "alpha",
+  "phaseText": "Service phase text goes here",
+  "serviceEmailAddress": "form-builder.no-reply@digital.justice.gov.uk",
```
Comment (umaar) [inline metadata/config/service.json]: What's the `.no-reply` bit about? Is that just for the form user to see and understand they can't reply to the email?

File: specifications/header/data/valid/header.navigation.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  "_id": "header.navigation",
+  "_isa": "another.header",
+  "_type": "header",
+  "homepageUrl": "homepageUrl",
+  "serviceName": "serviceName",
+  "serviceUrl": "serviceUrl",
+  "navigation": {
+    "_id": "header.navigation.items",
+    "_type": "header.navigation",
+    "xshow": {
```
Comment (umaar) [inline specifications/header/data/valid/header.navigation.json]: `xshow`

**Review by umaar — APPROVED** [review]

File: specifications/header/data/valid/header.navigation.json (hunk 2 above)
Comment (solidgoldpig) [reply specifications/header/data/valid/header.navigation.json]: Removed. Good spot

File: metadata/config/service.json (hunk 1 above)
Comment (solidgoldpig) [reply metadata/config/service.json]: I've just reverted it to be our standard form-builder@digital.justice.gov.uk address in the absence of us actually having any idea what the email should be (or could be)

**Pull request merged**